src/utils/DecoyDatabase.cpp

```diff
@@ -86,10 +86,11 @@ class TOPPDecoyDatabase :
     setValidFormats_("in", ListUtils::create<String>("fasta"));
     registerOutputFile_("out", "<file>", "", "Output FASTA file where the decoy database will be written to.");
     setValidFormats_("out", ListUtils::create<String>("fasta"));
-    registerStringOption_("decoy_string", "<string>", "_rev", "String that is appended to the accession of the protein database to indicate a decoy protein.", false);
-    registerStringOption_("decoy_string_position", "<enum>", "suffix", "Should the 'decoy_string' be prepended (prefix) or appended (suffix) to the protein accession?", false);
+    registerStringOption_("decoy_string", "<string>", "dec_", "String that is appended to the accession of the protein database to indicate a decoy protein.", false);
```

Comment:
`.. that is *combined* with the accession of the protein *identifier*...`
 (fixes in *)